apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Adapter.kt

```diff
@@ -81,47 +81,3 @@ interface Adapter<T> {
    */
   fun toJson(writer: JsonWriter, customScalarAdapters: CustomScalarAdapters, value: T)
 }
-
-fun <T> Adapter<T>.toJson(
-    value: T,
-    customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
-    indent: String = "",
-): String {
-  val buffer = Buffer()
-
-  toJson(buffer, value, customScalarAdapters, indent)
-  return buffer.readUtf8()
-}
-
-fun <T> Adapter<T>.toJson(
-    sink: BufferedSink,
-    value: T,
-    customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
-    indent: String = "",
-) {
-  val writer = BufferedSinkJsonWriter(sink)
-  writer.indent = indent
-
-  toJson(writer, customScalarAdapters, value)
-}
-
-fun <T> Adapter<T>.fromJson(
-    bufferedSource: BufferedSource,
-    customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
-): T {
-  return fromJson(BufferedSourceJsonReader(bufferedSource), customScalarAdapters)
-}
-
-fun <T> Adapter<T>.fromJson(
-    string: String,
-    customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
-): T {
-  return fromJson(Buffer().apply { writeUtf8(string) }, customScalarAdapters)
-}
-
-fun <T, M : Map<String, Any?>> Adapter<T>.fromMap(
-    map: M,
-    customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
-): T {
-  return fromJson(MapJsonReader(map), customScalarAdapters)
-}
```

Comment:
These were not used and also duplicating some of the actual interface methods. If we need helpers to adapt to//from a BufferedSource or something else, we can always add them back with better ergonomics

Reply:
Removed in https://github.com/apollographql/apollo-android/pull/3598/commits/4c8e9a9c753f4f180cd4a33465b2f98fcf77aad4

Reply:
Alright, there's more to this story, I reverted the removal, will do a different follow up PR